src/Page/Explore.elm

```diff
@@ -110,7 +111,7 @@ init scope dataset session =
       , tableState = SortableTable.initialSort initialSort
       }
     , session
-    , Cmd.none
+    , Ports.scrollTo { x = 0, y = 0 }
```

Comment:
Careful with this as this will scroll to page top even when a details modal is opened, losing user's scroll position with the table behind (meaning poor UX)

Reply:
I have no good suggestion, hence why it hasn't been fixed in 4 years 😅